public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -234,7 +234,7 @@ export function EditCloudGroupModal(props: ModalProps): React.ReactElement {
     handleSubmit,
     register,
     watch,
-    formState: { isDirty, errors },
+    formState: { isDirty, errors, isValid },
```

Comment:
Strangely, just destructuring the `isValid` is enough to trigger `act` warnings in the tests for this component - even if it isn't used 🤷 
I haven't dived into why as I don't have the chance to right now, but I updated the tests regardless to wait for the component to settle before asserting

Reply:
🤔

Reply:
something to investigate when we have time 😊